Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -42,3 +42,13 @@ trait-name-Thieving = Thieving
 trait-description-Thieving =
     You are deft with your hands, and talented at convincing people of their belongings.
     You can identify pocketed items, steal them quieter, and steal ~33% faster.
+
+trait-name-ForeignerLight = Foreigner (light)
```

Comment:
```suggestion
trait-name-ForeignerLight = New in Town
```

Don't really like the Light appended to the end; this should differentiate them enough I guess, makes some sense too?

Shrug

Reply:
Ideally this would also change the Loc string but I'm on mobile and don't want to go looking for every instance manually....